roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1303,7 +1304,7 @@ def get_defaults(self, roles):
 
         common = dict(use_openshift_sdn=True, ip=ip_addr, public_ip=ip_addr,
                       deployment_type='origin', hostname=hostname,
-                      public_hostname=hostname)
+                      public_hostname=hostname, portal_net='172.30.0.0/16')
```

Comment:
does ansible have a variable to store portal_net or do we have to hard-code this.